### PR TITLE
Feature/travis

### PR DESCRIPTION
**Pull request by sdepold**
boom

### CODE REVIEW

File: apiary/Makefile [hunk 1]
```diff
@@ -52,9 +52,9 @@ publish: test
 	apiary publish --api-name=$(IMAGES_APIARY_NAME) --path=$(IMAGES_BLUEPRINT)
 
 test: $(CDA_BLUEPRINT) $(CPA_BLUEPRINT) $(CMA_BLUEPRINT)
-	dredd $(CPA_BLUEPRINT) $(CPA_HOST)
-	dredd $(CDA_BLUEPRINT) $(CDA_HOST)
-	dredd $(CMA_BLUEPRINT) $(CMA_HOST) \
+	node_modules/.bin/dredd $(CPA_BLUEPRINT) $(CPA_HOST)
+	node_modules/.bin/dredd $(CDA_BLUEPRINT) $(CDA_HOST)
+	node_modules/.bin/dredd $(CMA_BLUEPRINT) $(CMA_HOST) \
```
Comment (johanneswuerbach) [inline apiary/Makefile]: Maybe `npm run dredd -- $(CPA_BLUEPRINT) $(CPA_HOST)` and add `dredd` to scripts.

Comment (sdepold) [inline apiary/Makefile]: fixed it in make land

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+language: node_js
+
+node_js:
+  - iojs-v2
+
+before_install:
+  - cd apiary
+  - make install
```
Comment (neonichu) [inline .travis.yml]: we should rather do a `make -C apiary install`

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+script:
+  - make test
```
Comment (neonichu) [inline .travis.yml]: we should rather do a `make -C apiary test`

File: .travis.yml (hunk 2 above)
Comment (sdepold) [inline .travis.yml]: :+1:

**Pull request merged**